indico/modules/events/requests/controllers.py

```diff
@@ -122,18 +120,13 @@ def process_form(self):
         else:
             req = self.request
             new = False
-        try:
-            self.definition.send(req, form.data)
-        except RequestModuleError:
```

Comment:
I would keep this. Could be useful for others implementing requests that need to call external services.